packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGrid-test.js

```diff
@@ -70,7 +70,7 @@ describe('BpkCalendarScrollGrid', () => {
           formatDateFull={formatDateFull}
           DateComponent={BpkCalendarScrollDate}
           daysOfWeek={weekDays}
-          weekStartsOn={3}
+          weekStartsOn={5}
```

Comment:
Why did we change this?

Reply:
@k0nserv there were some issues with updating snapshots, so I updated the tests